recipes/libjpeg-turbo/all/conanfile.py

```diff
@@ -89,11 +93,16 @@ def build_requirements(self):
     def source(self):
         tools.get(**self.conan_data["sources"][self.version])
         os.rename(self.name + "-" + self.version, self._source_subfolder)
+        if tools.cross_building(self.settings):
+            tools.replace_in_file(os.path.join(self._source_subfolder,"CMakeLists.txt"),"CMAKE_SYSTEM_PROCESSOR","CMAKE_SYSTEM_PROCESSOR_TARGET")
 
     def _configure_cmake(self):
         if self._cmake:
             return self._cmake
         self._cmake = CMake(self, set_cmake_flags=True)
+        if tools.cross_building(self.settings):
+            if self.settings.arch == "armv8":
+                self._cmake.definitions["CMAKE_SYSTEM_PROCESSOR_TARGET"] = "aarch64"
```

Comment:
not sure why the only armv8 is affected? maybe there is a wider conan issue? /cc @jgsogo

Reply:
I'm with @SpaceIm , I don't know what is CMAKE_SYSTEM_PROCESSOR_TARGET, I cannot find it in CMake docs.

Reply:
maybe it was supposed to be https://cmake.org/cmake/help/v3.1/variable/CMAKE_SYSTEM_PROCESSOR.html ? 🤔